PC/python3dll.c

```diff
@@ -342,6 +342,7 @@ EXPORT_FUNC(PyModule_AddFunctions)
 EXPORT_FUNC(PyModule_AddIntConstant)
 EXPORT_FUNC(PyModule_AddObject)
 EXPORT_FUNC(PyModule_AddStringConstant)
+EXPORT_FUNC(PyModule_AddType)
```

Comment:
You may move these changes into a separated PR, since these changes are not controversial.

Reply:
They won't really help without the METH_METHOD calling convention, so I'll keep them here :)